src/components/editor/ResourceProperty.jsx

```diff
@@ -39,11 +44,12 @@ export class ResourceProperty extends Component {
         newReduxPath.push(rtProperty.propertyURI)
         const payload = { reduxPath: newReduxPath, property: rtProperty }
         this.props.initNewResourceTemplate(payload)
-
+        const isAddDisabled = rtProperty.repeatable == "false" ? false: true
```

Comment:
So if repeatable is false, we want to say add is *not* disabled? The other checks above do the opposite.

Reply:
(see my comment above)

Reply:
(see all four cases indicated in recent comment above)

Reply:
See latest commit using new `isRepeatable` function from Utilities.